spec/lib/gitlab/closing_issue_extractor_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+describe Gitlab::ClosingIssueExtractor do
+  let(:project) { create(:project) }
+  let(:issue) { create(:issue, project: project) }
+
+  before do
+    @iid1 = issue.iid
```

Comment:
Can this be changed to `let(:iid1) { issue.iid }`? I think it would be better than mixing the `let` syntax and class variables.